polkadot/node/core/pvf/common/build.rs

```diff
@@ -0,0 +1,28 @@
+fn main() {
+	get_wasmtime_version();
+}
+
+pub fn get_wasmtime_version() {
```

Comment:
Maybe add a comment that in case there are multiple wasmtime versions being used, is an error. Example:

```
 $ cargo tree --package=aead --depth=0
error: There are multiple `aead` packages in your project, and the specification `aead` is ambiguous.
Please re-run this command with one of the following specifications:
  aead@0.3.2
  aead@0.4.3
  aead@0.5.2
```

I think an error in that case great, we definitely don't want to be mixing wasmtime versions anyway.

Reply:
I mean a comment would be useful for readers of the code. I was wondering what would happen if there were multiple versions of wasmtime, I didn't know it would be an error until I tried it.